Tribler/Policies/BoostingManager.py

```diff
@@ -0,0 +1,676 @@
+# -*- coding: utf-8 -*-
+# Written by Egbert Bouman, Mihai Capotă, Elric Milon, and Ardhi Putra Pratama H
+"""Manage boosting of swarms"""
+import ConfigParser
+import json
+import logging
+import os
+import random
+from binascii import hexlify, unhexlify
+
+import errno
+import libtorrent as lt
+
+from Tribler.Core.DownloadConfig import DownloadStartupConfig
+from Tribler.Core.TorrentDef import TorrentDef
+from Tribler.Core.Utilities.install_dir import determine_install_dir
+from Tribler.Core.simpledefs import DLSTATUS_SEEDING, NTFY_TORRENTS, NTFY_UPDATE, NTFY_CHANNELCAST
+from Tribler.Main.globals import DefaultDownloadStartupConfig
```

Comment:
If we could move `DefaultDownloadStartupConfig` to the core, we can move the `BoostingManager` to the core.